code/modules/spells/spell_types/godhand.dm

```diff
@@ -109,3 +109,54 @@
 	M.Stun(40)
 	M.petrify()
 	return ..()
+
+
+/obj/item/melee/touch_attack/duffelbag
+	name = "\improper burdening touch"
+	desc = "Where is the bar from here?"
+	catchphrase = "BA'R A'RP!!"
+	on_use_sound = 'sound/magic/disintegrate.ogg'
+	icon_state = "disintegrate"
+	inhand_icon_state = "disintegrate"
```

Comment:
Recolour a beige icon_state like you said.

Reply:
I went with a nice piss yellow, you're welcome